### PR TITLE
549 Updated product category list for opco home

### LINKED ISSUE #549: [Product Opco Home] Update UI block product category list

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #549 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://549-ui-product-category-list--danaher-ls-aem--hlxsites.hlx.page/us/en/products/brands/opco-home

### CODE REVIEW

File: blocks/product-category-list/product-category-list.js [hunk 1]
```diff
@@ -22,21 +22,45 @@ function createCard(product, firstCard = false) {
...
+  let isbrandPage = false;
+  isbrandPage = (block.classList.contains('opco-home'));
+  const metaBrand = getMetadata('brand');
   let products = await ffetch('/us/en/products-index.json')
     .filter(({ fullCategory }) => fullCategory.split('|').length === 1)
     .filter(({ fullCategory }) => fullCategory !== '')
     .filter(({ path }) => !path.includes('/topics'))
     .filter(({ path }) => !path.includes('/product-coveo'))
     .filter(({ path }) => !path.includes('/brands'))
+    .filter(({ brand }) => {
```
Comment (mhaack) [inline blocks/product-category-list/product-category-list.js:60]: I think the path filters above should be simpler with a category type filter. This should exclude topics, brands etc. already.

Comment (rgravitvl) [reply blocks/product-category-list/product-category-list.js:58]: Done

File: blocks/product-category-list/product-category-list.js [hunk 2]
```diff
@@ -22,21 +22,45 @@ function createCard(product, firstCard = false) {
...
   }, cardWrapper);
   card.querySelector('img').className = 'mb-2 h-24 w-full object-cover';
   return card;
 }
 
 export default async function decorate(block) {
   block.parentElement.parentElement.classList.add('!pb-0');
-
+  let isbrandPage = false;
+  isbrandPage = (block.classList.contains('opco-home'));
```
[the remark this answers is not in the document]
Comment (rgravitvl) [reply blocks/product-category-list/product-category-list.js]: Done

**Review by mhaack — APPROVED** [review]

**Review by davenichols-DHLS — APPROVED** [review]
Looks great, I can only see the subset of data.  Thanks.

**Pull request merged**